src/commands/registerCommands.ts

```diff
@@ -47,45 +47,45 @@ import { swapSlots } from './swapSlots';
+    registerCommandWithTreeNodeUnwrapping('appService.Delete', deleteWebApp);
+    registerCommandWithTreeNodeUnwrapping('appService.DeploymentScript', generateDeploymentScript);
+    registerCommandWithTreeNodeUnwrapping('appService.DisconnectRepo', disconnectRepo);
+    registerCommandWithTreeNodeUnwrapping('appService.EnableFileLogging', enableFileLogging);
+    registerCommandWithTreeNodeUnwrapping('appService.InstallCosmosDBExtension', installCosmosDBExtension);
+    registerCommandWithTreeNodeUnwrapping('appService.LoadMore', async (actionContext: IActionContext, node?: AzExtTreeItem) => await ext.rgApi.tree.loadMore(nonNullValue(node), actionContext));
+    registerCommandWithTreeNodeUnwrapping('appService.openFile', showFile, 500);
+    registerCommandWithTreeNodeUnwrapping('appService.OpenInPortal', openInPortal);
+    registerCommandWithTreeNodeUnwrapping('appService.Refresh', async (actionContext: IActionContext, node?: AzExtTreeItem) => await ext.rgApi.tree.refresh(actionContext, node));
+    registerCommandWithTreeNodeUnwrapping('appService.RemoveCosmosDBConnection', removeCosmosDBConnection);
+    registerCommandWithTreeNodeUnwrapping('appService.Restart', restartWebApp);
+    registerCommandWithTreeNodeUnwrapping('appService.RevealConnection', revealConnection);
+    registerCommandWithTreeNodeUnwrapping('appService.RevealConnectionInAppSettings', revealConnectionInAppSettings);
+    registerCommandWithTreeNodeUnwrapping('appService.ScaleUp', async (_context: IActionContext, node?: DeploymentSlotsNATreeItem | ScaleUpTreeItem) => await uiOpenInPortal(nonNullValue(node), nonNullValue(node).scaleUpId));
+    registerCommandWithTreeNodeUnwrapping('appService.showOutputChannel', () => { ext.outputChannel.show(); });
+    registerCommandWithTreeNodeUnwrapping('appService.Start', startWebApp);
+    registerCommandWithTreeNodeUnwrapping('appService.StartRemoteDebug', startRemoteDebug);
+    registerCommandWithTreeNodeUnwrapping('appService.StartSsh', startSsh);
+    registerCommandWithTreeNodeUnwrapping('appService.startStreamingLogs', startStreamingLogs);
+    registerCommandWithTreeNodeUnwrapping('appService.Stop', stopWebApp);
+    registerCommandWithTreeNodeUnwrapping('appService.StopLogStream', stopStreamingLogs);
+    registerCommandWithTreeNodeUnwrapping('appService.SwapSlots', swapSlots);
+    registerCommandWithTreeNodeUnwrapping('appService.toggleAppSettingVisibility', async (actionContext: IActionContext, node?: AppSettingTreeItem) => { await nonNullValue(node).toggleValueVisibility(actionContext); }, 250);
+    registerCommandWithTreeNodeUnwrapping('appService.ViewCommitInGitHub', viewCommitInGitHub);
+    registerCommandWithTreeNodeUnwrapping('appService.ViewProperties', viewProperties);
     registerSiteCommand('appService.Deploy', deploy);
     registerSiteCommand('appService.DeploySlot', deploySlot);
     registerSiteCommand('appService.Redeploy', redeployDeployment);
```

Comment:
We need to apply unwrapping to these commands as well. I've added a utility (`unwrapArgs`) to accomplish this in https://github.com/microsoft/vscode-azuretools/pull/1246.